renderer/components/editor/options/right.js

```diff
@@ -11,6 +11,13 @@ class RightOptions extends React.Component {
     const formatOptions = options ? options.map(({format, prettyFormat}) => ({value: format, label: prettyFormat})) : [];
     const pluginOptions = options ? options.find(option => option.format === format).plugins.map(plugin => ({value: plugin.title, label: plugin.title})) : [];
 
+    if (pluginOptions.length < 2) {
+      pluginOptions.push({
```

Comment:
I think we should have a separator here.

Reply:
Seems like there's no native way to add a separator to an html select. I used [this](https://stackoverflow.com/questions/899148/html-select-option-separator) and it shows up like this:
![Screenshot 2019-07-08 11 53 27](https://user-images.githubusercontent.com/8822835/60824168-1f4f9480-a177-11e9-839d-8d6ca2ab989e.png)

Ideally, I would change that to use the electron menu, but that would be a bit more of a refactor, since that component is used in a couple of places

Reply:
Hmm, that doesn't look very good. We can just skip the separator until we use the Electron API.